### DISCUSSION TITLE
Experiences with installing ARC into multiple clusters

### DISCUSSION THREAD

**Discussion by jimmyh85** [actions/actions-runner-controller#1359] [General]
Has anyone installed the ARC into multiple clusters already? I wonder especially, if I we would have to configure a webhookserver into each cluster when using webhook driven autoscaling?

A solution with a central webhookserver which communicates with the runner controllers would probably be preferred, but I think that does not work currently. 

Thank you

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
